lib/lte_link_control/lte_lc.c

```diff
@@ -465,13 +465,6 @@ static int init_and_config(void)
 	LOG_INF("PDN Auth: %s", log_strdup(cgauth));
 #endif
 
-	/* Listen for RRC connection mode notifications */
-	err = enable_notifications();
-	if (err) {
-		LOG_ERR("Failed to enable notifications");
-		return err;
-	}
-
```

Comment:
Yes, it seemed redundant to me to have it here as well. Couldn't think of a use case where one wants to initialize the link controller without using its API to also control the functional mode. It may very well be that I missed a valid use case, so I can put it back, it does no harm to keep it.

Reply:
Yeah, no harm to be there just for sure, but yeah might be rare if there are clients that want to have notifications from api but are controlling modem by using at commands.

Reply:
I've put it back, just in case